lua/autorun/client/cl_devtools.lua

```diff
@@ -0,0 +1,133 @@
+local ranOnce = !ranOnce
+if (!ranOnce) then return end -- just to prevent edge cases
+ranOnce = true
+
+local isDev = isDev
+local refreshTime = 2
+local rootFolder = rootFolder or nil
+local fileTimes = fileTimes or {}
+local devReplStr = nil -- classname of all child files, taken from folder
+
+local function replaceFunction(base, regex, replacement)
+	local iStart, iEnd = string.find(base, regex, 1)
+
+	while iStart do
+		local xStart, _ = string.find(base, "(", iEnd, true) -- func name index
+		base = string.sub(base, 1, iStart-1) ..
+			replacement .. "." .. string.sub(base, iEnd + 1, xStart-1) ..
+			" = function" .. string.sub(base, xStart)
+		iStart, iEnd = string.find(base, regex, xStart)
+	end
+	return base
+end
+
+local function refreshFile(path)
+	if (!file.Exists(path, "LUA")) then return print("File " .. path .. " doesn't exist") end
+	if (!isDev) then return end
```

Comment:
Why check this? Structure your code so you know it's not being called in the first place.